util/config/config.go

```diff
@@ -24,12 +24,14 @@ const (
 
 const (
 	MaxNumTxnPerBlock   = 4096
+	MaxBlockSize        = 1 * 1024 * 1024
```

Comment:
We should add a comment about the unit (bit/byte) or rename the variable to include the unit